contracts/wallet.sol

```diff
@@ -768,7 +769,8 @@ contract Wallet is ENSResolvable, Vault, GasTopUpLimit, LoadLimit {
     event UpdatedAvailableLimit();
 
     string constant public WALLET_VERSION = "3.1.0";
-    uint constant private _DEFAULT_MAX_STABLECOIN_LOAD_LIMIT = 10000; //10,000 USD
+    uint constant private _MIN_STABLECOIN_LOAD_LIMIT = 1;     // 1 USD
```

Comment:
In theory, this could be 0 as in setSpendLimit().

Reply:
Yes, let's do this @i-stam ... If it keeps us inline with the other limits which we use.

Reply:
@mischat should I go for it, in theory it reverts all the PR changes